src/core/lib/security/credentials/ssl/ssl_credentials.cc

```diff
@@ -66,12 +68,40 @@ grpc_ssl_credentials::create_security_connector(
   absl::optional<std::string> overridden_target_name =
       args->GetOwnedString(GRPC_SSL_TARGET_NAME_OVERRIDE_ARG);
   auto* ssl_session_cache = args->GetObject<tsi::SslSessionLRUCache>();
+  tsi_ssl_session_cache* tsi_ssl_session_cache =
+      ssl_session_cache == nullptr ? nullptr : ssl_session_cache->c_ptr();
+
+  const tsi_ssl_root_certs_store* root_store;
+  // Use default root certificates.
```

Comment:
micro-nit: Maybe this should say "Use the default (e.g. OS) root certificates if the user did not pass in any root certificates."